multi-node/vagrant/Vagrantfile

```diff
@@ -91,13 +92,13 @@ Vagrant.configure("2") do |config|
 
   ["vmware_fusion", "vmware_workstation"].each do |vmware|
     config.vm.provider vmware do |v|
-      v.vmx['numvcpus'] = 1
+      v.vmx['numvcpus'] = 2
       v.gui = false
     end
   end
 
   config.vm.provider :virtualbox do |vb|
-    vb.cpus = 1
+    vb.cpus = 2
```

Comment:
I had run into some stability issues with one, but that may have been while I was trying to run the policy controller on the master, I'll retest and and revert back if possible.